runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -197,36 +199,47 @@ default boolean sendNotificationsWhenFocused()
 	}
 
 	@ConfigItem(
-		keyName = "fontType",
-		name = "Dynamic Overlay Font",
+		keyName = "clientFont",
+		name = "Client Font",
+		description = "Configures what font type is used for the client",
+		position = 26
+	)
+	default Font clientFont()
+	{
+		return FontManager.getRunescapeFont();
+	}
+
+	@ConfigItem(
+		keyName = "overlayGroup1Font",
+		name = "Overlays 1",
 		description = "Configures what font type is used for in-game overlays such as player name, ground items, etc.",
-		position = 30
+		position = 27
 	)
-	default FontType fontType()
+	default Font overlayGroup1Font()
 	{
-		return FontType.SMALL;
+		return FontManager.getRunescapeSmallFont();
 	}
 
 	@ConfigItem(
-		keyName = "tooltipFontType",
-		name = "Tooltip Font",
-		description = "Configures what font type is used for in-game tooltips such as food stats, NPC names, etc.",
-		position = 31
+		keyName = "overlayGroup2Font",
+		name = "Overlays 2",
+		description = "Configures what font type is used for in-game overlays such as boost info, opponent info and plugin specific overlays",
+		position = 28
 	)
-	default FontType tooltipFontType()
+	default Font overlayGroup2Font()
 	{
-		return FontType.SMALL;
+		return FontManager.getRunescapeFont();
 	}
 
 	@ConfigItem(
-		keyName = "interfaceFontType",
-		name = "Interface Overlay Font",
-		description = "Configures what font type is used for in-game interface overlays such as panels, opponent info, clue scrolls etc.",
-		position = 32
+		keyName = "overlayGroup3Font",
```

Comment:
Looks like you need to rebase this config file. Shouldn't need to change every config message and name.

Reply:
This is intentional and not due to rebase. The old names weren't really accurate anymore. At first i based the font selection on tooltip positioning but this didn't really work out because 99% has position top-left/dynamic positioning while they definitely should have the option to select different fonts. I generalized it to groups and added descriptions to tell the user which group has what kind of components. Overlay x isn't really i good name, i realize that, but you can only really fit 1 word in the name or it will be cut off in the settings screen.